src/main/java/xogame/game_over/GameOver.java

```diff
@@ -13,9 +13,13 @@ public static Boolean isAnyRowComplete(Board board) {
         return doesAnyInnerListMatchTheCondition(board.getRows(), listPredicate);
     }
 
+    public static Boolean isAnyColumnComplete(Board board) {
+        return doesAnyInnerListMatchTheCondition(board.getColumns(), listPredicate);
+    }
+
     public static Boolean doesAnyInnerListMatchTheCondition(List<List<Integer>> listOfLists, Predicate<List<Integer>> condition) {
         List<List<Integer>> listOfListsWithoutEmptyLists = listOfLists.stream().filter(e -> !e.isEmpty()).collect(Collectors.toList());
```

Comment:
I do not understand why we are removing the empty lines and collecting the result into a list to `stream()` it right after that. Can you explain that?